ports/nrf/boards/feather52840/pins.csv

```diff
@@ -0,0 +1,48 @@
+P0_00,P0_00,PA0
```

Comment:
I think you misunderstood me, the ",PA0" can be removed, just leave "P0_00,P0_00" :)